app/src/sailfish-qml/qml/soundcloud/SoundCloudTrackPage.qml

```diff
@@ -0,0 +1,24 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import harbour.musikloud2.MusiKloud 2.0
+import QtGraphicalEffects 1.0
```

Comment:
No we don't need this anymore.
QtGraphicalEffects supports some fancy effects ie Shadows should the black frame for the position switch in the player UI be to harsh. Currently not in focus though.